SofaKernel/modules/SofaBaseCollision/src/SofaBaseCollision/MinProximityIntersection.cpp

```diff
@@ -63,12 +62,23 @@ void MinProximityIntersection::init()
 {
     intersectors.add<CubeCollisionModel, CubeCollisionModel, MinProximityIntersection>(this);
     intersectors.add<SphereCollisionModel<sofa::defaulttype::Vec3Types>, SphereCollisionModel<sofa::defaulttype::Vec3Types>, MinProximityIntersection>(this);
+    intersectors.add<RigidSphereModel,RigidSphereModel, MinProximityIntersection> (this);
+    intersectors.add<SphereCollisionModel<sofa::defaulttype::Vec3Types>, RigidSphereModel, MinProximityIntersection>(this);
 
     IntersectorFactory::getInstance()->addIntersectors(this);
 
 	BaseProximityIntersection::init();
 }
 
+bool MinProximityIntersection::testIntersection(Cube& cube1, Cube& cube2)
+{
+    return BaseProximityIntersection::testIntersection(cube1, cube2);
+}
+
+int MinProximityIntersection::computeIntersection(Cube& cube1, Cube& cube2, OutputVector* contacts)
+{
+    return BaseProximityIntersection::testIntersection(cube1, cube2);
```

Comment:
@fredroy do you confirm the typo here? Shouldn't it be `BaseProximityIntersection::computeIntersection`?

Reply:
I do confirm it should be computeIntersection...

Reply:
Ok, I'm on it